x/rollapp/keeper/block_height_to_finalization_queue_test.go

```diff
@@ -81,137 +83,196 @@ func TestBlockHeightToFinalizationQueueGetAll(t *testing.T) {
 	)
 }
 
-// TODO: Test FinalizeQueue function with failed states
 func (suite *RollappTestSuite) TestFinalizeRollapps() {
-	type stateUpdate struct {
-		blockHeight int64
-		startHeight uint64
-		numBlocks   uint64
+	suite.SetupTest()
+
```

Comment:
In testify I dont think you have to call this expliclity

Reply:
you do